test/lib/ansible_test/_util/controller/sanity/validate-modules/validate_modules/schema.py

```diff
@@ -542,7 +549,7 @@ def list_dict_option_schema(for_collection, plugin_type):
     basic_option_schema = {
         Required('description'): doc_string_or_strings,
         'required': bool,
-        'choices': list,
+        'choices': Any(list, {any_string_types: doc_string_or_strings}),
```

Comment:
```suggestion
        'choices': Any(list, {object: doc_string_or_strings}),
```
That way you can use dicts for choices also for other key types than strings.

Reply:
(I thought I submitted this comment already during the weekend, but apparently I did not...)

Reply:
i swear i tried this iteration before ... but .. .passes now.. not looking back